examples/events/interactionEvents/index.js

```diff
@@ -5,7 +5,8 @@ var nodes = new vis.DataSet([
   { id: 2, label: 'Node 2', title: 'I have a popup!' },
   { id: 3, label: 'Node 3', title: 'I have a popup!' },
   { id: 4, label: 'Node 4', title: 'I have a popup!' },
-  { id: 5, label: 'Node 5', title: 'I have a popup!' }
+  { id: 5, label: 'Node 5', title: 'I have a popup!' },
+  { id: 6, label: 'Node 6', title: 'I have a popup!' }
```

Comment:
@Tav0, so I tested this new example data on master (without the fix) and I am not able to reproduce the bug.  can you tell me more about when you see the bug (see hover broken)? 

Could you perhaps add another example to the PR that will show the bug if the fix is not present?

Reply:
some more details on what I tried in this issue https://github.com/visjs-community/visjs-network/issues/51